source/index.md

```diff
@@ -1,11 +1,9 @@
-# The Island Dispatch 🏝️
-
-Here's the napari community blog!
-
-## All posts
+# Welcome to the napari community blog! 🏝️
 
 ```{postlist}
    :list-style: circle
-   :format: "{title}"
-   :sort:
+   :format: {title} | {date} | {author}
```

Comment:
it's a big improvement, but very wall of text.
Can we try something like:
```suggestion
   :format: \n{title}\nby {author}\n{date}
```
not certain quotes are needed.